File: DSA/Gautam_Shyamnani_590013078/Day15Question1.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    int arr[] = {64, 34, 25, 12, 22, 11, 90};
    int n = sizeof(arr) / sizeof(arr[0]);

    sort(arr, arr + n);

    for (int i = 0; i < n; i++) {
        cout << arr[i] << " ";
    }
    return 0;
}

File: DSA/NiharikaChand_590017697/Day15Question1.java
import java.util.*;
class Day15Question1
{
    public static void main(String arg[]) 
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter size of array: ");
        int n= sc.nextInt();
        int arr[]=new int[n]; int i;

        for(i=0; i<n; i++)
        {
            System.out.println("Enter an element: ");
            arr[i]=sc.nextInt();
        }

        Arrays.sort(arr);
        System.out.println("Sorted array: ");
        for(i=0; i<n;i++)
        {
            System.out.print(arr[i]+" ");
        }
    }
}